etc/picongpu/taurus-tud/k80_picongpu.profile.example

```diff
@@ -73,6 +77,7 @@ export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
 export TBG_SUBMIT="sbatch"
 export TBG_TPLFILE="etc/picongpu/taurus-tud/k80.tpl"
 
+alias getDevice='srun -p gpu2-interactive --gres=gpu:0 --ntasks=1 --pty --mem=8G -t 8:00:00 bash'
```

Comment:
Why `--gres=gpu:0`, shouldn't it be 1 GPU?

Reply:
all devices should be `--gres=gpu:8` for K80 but maybe zero is take all you have?

Reply:
That's a function to get 1 device, not a whole node

Reply:
I copied it from @steindev's script https://gist.github.com/steindev/86df43bef49586e2b33d2fb0a372f09c. When I was looking over the PR I also wondered about it... But it works for me currently without any GPUs, so I wasn't sure if I'm supposed to change it or not.

Reply:
I will update it and force-push it

Reply:
Updated